### ISSUE TITLE
Support agent cannot transfer a ticket

### ISSUE THREAD

**Issue by wptech68** [nanodesigns/nanosupport#35]
Just a suggestion.

I think that a support agent would be able to transfer the ticket to another support agent, this will help tickets that need more than 1 department assistance to be resolved.
currently , only admin can assign a ticket, so if the current support agent finishes his help and need  to transfer it to another agent to complete the task, only the admin can do it.

perhaps, the other way is to assign more than 1 agent to a ticket, so it works like a conversation with multiple participants, one end user and 2 or more agents.

Hope it helps

**Comment by mayeenulislam** [comment]
Yes, we have a plan to incorporate multiple support agent feature, with a supportive plugin or by core.
But it will need some time to reOrganize code base, the way it _is_.
BTW, thank you for your proposal.

**Comment by TotusUnus** [comment]
This would be great to have it so that the notification email can be sent to more than one email address. This way if one agent is down then at least another agent will be notified and then login and reply to the ticket.
Great Plugin
